e2e/tests/oai/002.sh

```diff
@@ -27,18 +27,26 @@ source "${LIBDIR}/k8s.sh"
 # shellcheck source=e2e/lib/kpt.sh
 source "${LIBDIR}/kpt.sh"
 
+#adding directory path to register repository to have upstream packages in repository subdirectory when used with packagevariantset
+#TODO: to change the private catalog repository to nephio catalog repository after PR approval
+kpt alpha repo register --namespace default https://github.com/josephthaliath/catalog.git --directory=workloads/oai --name=oai-ran-packages
+
 k8s_apply "$TESTDIR/002-oai-operators.yaml"
 
-for pkgvar in common-core-database cp-operators up-operators; do
+for pkgvar in common-core-database cp-operators up-operators ran-operator-edge-oai-ran-operator ran-operator-regional-oai-ran-operator; do
     k8s_wait_ready "packagevariant" "oai-$pkgvar"
 done
 kpt_wait_pkg "core" "database"
 kpt_wait_pkg "core" "oai-cp-operators"
 kpt_wait_pkg "edge" "oai-up-operators"
+kpt_wait_pkg "regional" "oai-ran-operator"
+kpt_wait_pkg "edge" "oai-ran-operator"
 
 _core_kubeconfig="$(k8s_get_capi_kubeconfig "core")"
 k8s_wait_ready_replicas "deployment" "mysql" "$_core_kubeconfig" "oai-core"
 for controller in amf ausf nrf smf udm udr; do
     k8s_wait_ready_replicas "deployment" "oai-$controller-controller" "$_core_kubeconfig" "oai-operators"
 done
 k8s_wait_ready_replicas "deployment" "oai-upf-controller" "$(k8s_get_capi_kubeconfig "edge")" "oai-operators"
+k8s_wait_ready_replicas "deployment" "oai-ran-operator" "$(k8s_get_capi_kubeconfig "regional")" "oai-ran-operators"
+k8s_wait_ready_replicas "deployment" "oai-ran-operator" "$(k8s_get_capi_kubeconfig "edge")" "oai-ran-operators"
```

Comment:
Maybe we can store the value of kubeconfig of the edge and reuse it.
```suggestion
_edge_kubeconfig="$(k8s_get_capi_kubeconfig "edge")"
k8s_wait_ready_replicas "deployment" "oai-upf-controller" "$_edge_kubeconfig" "oai-operators"
k8s_wait_ready_replicas "deployment" "oai-ran-operator" "$(k8s_get_capi_kubeconfig "regional")" "oai-ran-operators"
k8s_wait_ready_replicas "deployment" "oai-ran-operator" "$_edge_kubeconfig" "oai-ran-operators"
```